app/backend/src/clj/teet/notification/notification_db.clj

```diff
@@ -58,8 +58,9 @@
       (meta-model/system-created))))
 
 (def notification-keys [:db/id :notification/status
-                        :notification/target :notification/type
+                        :notification/type
                         :meta/created-at :meta/creator
+                        {:notification/target [:db/id :file.part/name :file.part/number]}
```

Comment:
Aren't these keys are used for all types of notifications, but only some notifications will have file part info.
I don't think we have typically pulled notification type specific fields here.